cosmwasm/lib/wasmi-runtime/Enclave.config.xml

```diff
@@ -0,0 +1,12 @@
+<!-- Please refer to User's Guide for the explanation of each field -->
+<EnclaveConfiguration>
+  <ProdID>0</ProdID>
```

Comment:
in enigma-core the ProdID was 1. you should check what it means

Reply:
![Screenshot from 2020-04-08 00-40-51](https://user-images.githubusercontent.com/22766533/78722224-b4afcd00-7931-11ea-81cd-d565bf183ed5.png)
![Screenshot from 2020-04-08 00-40-59](https://user-images.githubusercontent.com/22766533/78722231-b7122700-7931-11ea-9e53-7899d2739227.png)

Source: [Intel_SGX_Developer_Reference_Linux_2.6_Open_Source.pdf](https://github.com/enigmampc/EnigmaBlockchain/files/4447128/Intel_SGX_Developer_Reference_Linux_2.6_Open_Source.pdf)

Reply:
Downloaded the pdf from https://download.01.org/intel-sgx/linux-2.6/docs/Intel_SGX_Developer_Reference_Linux_2.6_Open_Source.pdf

Reply:
It's our product number... We also have an ISV number from Intel we should put in this file.

Reply:
In core it was 1 and in km it was 2 https://github.com/enigmampc/enigma-core/blob/3c04c2625741daa693cdc1a867db03bc1dc3ead2/enigma-principal/enclave/Enclave.config.xml#L3

It's just our product number. We also have an ISV number from Intel that we should put there.